### PR TITLE
doc(package): Change meanjs version to 0.5.0-beta

### PR DESCRIPTION
**Pull request by codydaig**
Change the version to 0.5.0-beta so people know that this is beta code and things are rapidly changing.

### CODE REVIEW

**Comment by ilanbiala** [comment]
Seems fine to me I guess.

**Comment by rhutchison** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "meanjs",
   "description": "Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js.",
-  "version": "0.4.2",
-  "meanjs-version": "0.4.2",
+  "version": "0.5.0-beta",
```
Comment (ilanbiala) [inline package.json]: It doesn't really matter too much to me, the whole point is to just tag it as different so people know.

Comment (codydaig) [inline package.json]: I wanted to leave it generic on purpose. To me, RC1 indicates a specific release, where beta just says it's currently under development and changing.

Comment (lirantal) [inline package.json]: So go ahead and merge then, I don't see it as that much of an issue.

**Pull request merged**